### DISCUSSION TITLE
Tapping 2 notes stacked on top of eachother by alternating keys dont work

### DISCUSSION THREAD

**Discussion by Vortex100K** [ppy/osu#19753] [Q&A] [answered]
### Type

Game behaviour

### Bug description

so I was playing the new osu spotlight playlist and its very goofy with really slow patterns.

The first 2 maps have a lot of stacked notes at low AR so I could easily tell something was wrong 
every time I would alternate on these stacked notes I can only hit the note that appears first but the next note cant be hit 
I don't know if this has something to do with the new Notelock or if it may be my osu install but I cant hit them 

Single tapping these notes work fine 
I also tried the exact same map in osu stable and I could tap them with no problem 

here is the exact same map being played in both clients to the best of my ability 

osu Stable - https://user-images.githubusercontent.com/79509710/184502861-58582aa3-7897-45fe-9f5d-7cd785bc990b.mp4

osu lazer - https://user-images.githubusercontent.com/79509710/184502930-753027a3-e4cb-48dc-acb7-4b84dd36fe6d.mp4

### Version

2022.810.2

### Logs

[database.log](https://github.com/ppy/osu/files/9331832/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9331833/legacy-ipc.log)
[performance.log](https://github.com/ppy/osu/files/9331834/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9331835/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9331836/updater.log)

**Comment by peppy** [comment, marked as answer]
it sounds like your key bindings must be setup wrong. Can you take a screenshot of the osu! section of your keybinds and share which you are using in gameplay?

**Reply by Vortex100K** [reply]
![osu_2022-08-14_11-44-09](https://user-images.githubusercontent.com/79509710/184533181-d21880da-a010-4023-8d5e-b7011a3f6875.png)
omg im so stupid 😭 i set my binds like in osu stable ⚰ thanks peppy your an absolute legend!
-5 iq moment 😅
